modules/core/shared/src/main/scala/cron4s/Cron.scala

```diff
@@ -16,16 +16,14 @@
 
 package cron4s
 
-import scala.scalajs.js.annotation.JSExportTopLevel
 import scala.util.Try
 
-/**
-  * The entry point for parsing cron expressions
-  *
-  * @author Antonio Alonso Dominguez
-  */
-@JSExportTopLevel("Cron")
-object Cron {
+@FunctionalInterface
+trait Parser {
```

Comment:
nit: don't think we need the `@FunctionalInterface` annotation here, as it's meant for Java interaction only and don't expect anybody trying to implement their own parsers in Java, considering the return type is `Either`.

Also, for more FP-idiomatic style, the trait could be defined as:

```scala
type Parser = String => Either[Error, CronExpr]
```

Reply:
Well, I had a warning (promoted as compilation error on CI) asking me to put this annotation.
I can change it for a type definition if it matters.